roles/openshift_health_checker/openshift_checks/docker_image_availability.py

```diff
@@ -124,14 +129,18 @@ def local_images(self, images, task_vars):
         ]
 
     def is_image_local(self, image, task_vars):
-        result = self.module_executor("docker_image_facts", {"name": image}, task_vars)
+        """Check if image is already in local docker index."""
+        result = self.execute_module("docker_image_facts",
+                                     module_args={"name": image},
+                                     task_vars=task_vars)
```

Comment:
I don't think any module we didn't write actually needs task_vars. We also obviously don't need tmp in most places; the problem was that it was getting a dict for tmp in at least one case and trying to use it. Probably most modules never use that either, but I would've expected to notice for the one that did.